inc/hooks.php

```diff
@@ -49,3 +49,24 @@ function uds_wp_add_site_info() {
 		echo apply_filters( 'uds_wp_site_info_content', $site_info ); // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 	}
 }
+
+/**
+ * In Sept. 2010, The upstream Bootstrap replaced 'active' with 'is-active', but
+ * ONLY in sidebar menus. This filter searches the classes of a menu item, and
+ * replaces 'active' with 'is-active' - but only after checking to make sure this
+ * is a sidebar nav walker that is running.
+ */
+if ( ! function_exists( 'uds_wp_replace_active' ) ) {
+	function uds_wp_replace_active( $classes, $items, $args = '' ) {
+		if ( is_a( $args->walker, 'Uds_Custom_Walker_Widget_Nav_Menu' ) ) {
+			foreach( $classes as $key => &$val ) {
+				if ( 'active' === trim($val) ) {
+					$classes[$key] = 'is-active';
+				}
+			}
+		}
+
+		return $classes;
+	}
+}
+add_filter( 'nav_menu_css_class', 'uds_wp_replace_active', 20, 4);
```

Comment:
I don't think we need this function here in hooks.php, since we have created something similar on inc/navigation-widget.php
In **inc/navigation-widget.php**, you can change the line 44 to **is-active** and that should do the work.

```
/* Check if function exists. */
if ( ! function_exists( 'uds_wp_theme_special_nav_class' ) ) {
	/**
	 * Replace current-menu-item class with active class.
	 *
	 * @param classes $classes is an array of the CSS classes that are applied to the menu item's individual element.
	 *
	 * @param item    $item is the current menu item.
	 */
	function uds_wp_theme_special_nav_class( $classes, $item ) {
		if ( in_array( 'current-menu-item', $classes ) ) {
			$classes = array_diff( $classes, array( 'current-menu-item', 'active' ) );
			

                   $classes[] = 'active ';   <============ HERE - Change this to is-active



		} else if ( in_array( 'current_page_item', $classes ) ) {
			$classes = array_diff( $classes, array( 'current_page_item', 'active' ) );
			$classes[] = 'active ';
		}
		return $classes;
	}
}
```